targets/ath79-generic

```diff
@@ -0,0 +1,7 @@
+config 'CONFIG_GLUON_SPECIALIZE_KERNEL=y'
+config 'CONFIG_TARGET_SQUASHFS_BLOCK_SIZE=64'
```

Comment:
As we do not have memory-pressure in this target, these settings can be removed. We will get a better compressed rootfs and some kernel components built as a module.